pyproject.toml

```diff
@@ -58,7 +58,6 @@ document = [
 ]
 all = [
   "botorch",
-  "catalyst",
```

Comment:
Could you add a version constraint to fastai?

Reply:
Done in commit https://github.com/optuna/optuna-integration/pull/114/commits/07ed516dd27e97c4bf8d9c7be15ade08ca124f72 .